test/testNumericalAnalysis.cpp

```diff
@@ -11,13 +11,13 @@ SCENARIO("brent", "[numericalAnalysis][brent]") {
         const double parameter_) {
       return (parameter_ - 1.0) * std::pow(parameter_ + 1.0, 2.0) + 2.0;
     };
-    double objectiveFunctionRoot = (1.0 / 3.0) * (-1.0 - (4.0 / std::pow(19.0 - 3.0 * std::sqrt(33.0), 1.0 / 3.0)) - std::pow(19 - 3 * std::sqrt(33.0), 1.0 / 3.0));
+    double objectiveFunctionRoot = (1.0 / 3.0) * (-1.0 - (4.0 / std::pow(19.0 - 3.0 * std::sqrt(33.0), 1.0 / 3.0)) - std::pow(19.0 - 3.0 * std::sqrt(33.0), 1.0 / 3.0));
 
     const arma::uword numberOfIterations = discreteRandomNumber() + 100;
 
     WHEN("The root is between the bounds") {
       THEN("Return root value") {
-        const double lowerBound = -1.0 - objectiveFunctionRoot - std::abs(continuousRandomNumber());
+        const double lowerBound = objectiveFunctionRoot - 1.0 - std::abs(continuousRandomNumber());
         const double upperBound = 1.0 + std::abs(continuousRandomNumber());
```

Comment:
Ja tatsächlich. Die Zeile war noch ein Ableger von dem Versuch auf 100 Prozent Coverage zu kommen xD Ich fixe das dann ;) Danke 

-------- Ursprüngliche Nachricht --------
Von: Sebastian Niemann <notifications@github.com> 
Datum: 14.03.2016  20:38  (GMT+01:00) 
An: SebastianNiemann/Mantella <Mantella@noreply.github.com> 
Cc: OpusV <pusz@sra.uni-hannover.de> 
Betreff: Re: [Mantella] Fix: Unexcepted Brent exception (#187) 

In test/testNumericalAnalysis.cpp:
>  
>      const arma::uword numberOfIterations = discreteRandomNumber() + 100;
>  
>      WHEN("The root is between the bounds") {
>        THEN("Return root value") {
> -        const double lowerBound = -1.0 - objectiveFunctionRoot - std::abs(continuousRandomNumber());
> +        const double lowerBound = objectiveFunctionRoot - 1.0 - std::abs(continuousRandomNumber());
>          const double upperBound = 1.0 + std::abs(continuousRandomNumber());

Wäre es nicht sinnvoll hier dann

objectiveFunctionRoot + 1.0 + std::abs(continuousRandomNumber());

zu schreiben?

—
Reply to this email directly or view it on GitHub.